### DISCUSSION TITLE
Windows Terminal Preview v1.13.1098

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#12936] [Announcements]
This is an update to fix a number of issues identified in 1.13.1073x.

<details>
<summary>Why are there so many packages? How do I choose?</summary>
This version of Windows Terminal is distributed in two bundles, one of which works on Windows 10-11 and the other of
which only works on Windows 11. The Windows 11 version is much smaller because we no longer need to work around a
platform issue related to our dependencies.

If you intend on using Terminal as an unpackaged application--that is, extracting the `msix` file--we recommend that
you use the `Win10` bundle. You will need the [Visual C++ runtime redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).

In addition, if you install the _packaged_ version on either Windows 10 or Windows 11, it now depends on [the Visual C++ Universal Runtime Package](https://docs.microsoft.com/en-us/troubleshoot/developer/visualstudio/cpp/libraries/c-runtime-packages-desktop-bridge?msclkid=3e1dd82aac8b11ec8f9dbd86d5979ca8#how-to-install-and-update-desktop-framework-packages).

Despite these distributions having different version numbers, they are built from the same code and there is no
functional difference between them.

If you install the Windows 10 verison on Windows 11, it will probably automatically upgrade itself to the Windows 11
version. It turns out that it is impossible to have two bundles with _the same version number_, so it has to be this
way.

Since 1.12 is the final release that supports Windows 19H1, we're including a final update to 1.12 on the preview channel, just
so that the four of you stuck on that version get the latest fixes. 😄 
Install that manually at your own peril.
</details>

### Appearance

* Our Maximize/Restore button is now a fine round boi (#12660)

### Accessibility

* The profile list in the Settings UI now offers tooltips for long profile names (#12448)
* We'll automatically focus the window renamer textbox when it opens (#12798)
* High contrast will no longer result in a ridiculous and bad titlebar color (#12839)
* When you delete a color scheme, we'll move focus back to the color scheme list (#12841)
* Two instances of huge debug log spam with a screen reader connected have been stamped out (#12698) (#12723)

### Usability

* We've added some text to the color schemes page indicating that it is for editing--not setting--color schemes (#12663)
   * We're working to refine how color schemes are set and edited, so stay tuned for future improvements!
* The retro terminal effect (as well as other shaders) will now work on pre-D3D11 hardware! (#12677)
* Terminal will once again render properly when you move between different-DPI displays (#12713) (#12749)
* Resizing the window while a background color or underline is displayed will no longer smear it across the whole screen (#12637) plus a fix for a huge crash that PR introduced (#12853)
* It took us three releases to get it right, but we've finally solved the issue where we'd punch a hole straight through the Terminal when a dialog appeared (#12840)

### Reliability

* There was an issue on Windows 11 where Terminal would queue up billions of animations while the screen was off; it will now no longer do so (#12820)
* We've fixed crashes in `ProposeCommandline` (#12838), `Monarch::_GetPID` (#12856) and other parts of WT's RPC infrastructure (#12825)
* On Windows 10, the settings UI will no longer sometimes crash on close (we've updated to a new build of WinUI 2 for the fix!) (#12847)

### Miscellaneous

* Windows will no longer reject certain Terminal updates/reinstalls due to "differing package content" (#12779)
* Fragments can once again override the names of generated profiles (#12627)
* An issue from the 1073 series, where you could not upgrade the bundle using DISM, has been resolved (#12819)
   * As a result, our bundle version is now over three thousand!
* @dmezh contributed some wording changes to the text about transparency/opacity (#12592) (#12727) (thanks!)
* Some trailing commas that broke the JSON Schema document are no longer trailing, or present at all (#12644) (thanks @sowmya-hub!)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.13.10983.0'>Windows Terminal Preview v1.13.1098</a>.</em>

**Comment by sowmya-hub** [comment]
Great to see the new update and all the PRs merged! (UI improvements for accessibility seem very good) 
Thank you for being so helpful to newcomers like me!

**Reply by DHowett** [reply]
Hey, we couldn't do this without a great community like you all :) so thanks for the contribution!

**Reply by sowmya-hub** [reply]
It's great to be able to contribute and engage with the community, you're awesome for making it so inclusive :D

**Reply by btenden** [reply]
Fantastic program. I'm using it more and more.
The best is the bash-shortcuts crtl-shift-c and v. Finally on windows.

But there is a bug:
When using ssh, then tabtitle does not change and the title is "c:\windows\system32\conhost.exe" and the mouseover is "dwhacc c:\windows\system32\conhost.exe"

            {
                "commandline": "ssh dwh03.zorgnet.local",
                "guid": "{0caa0dad-35be-5f56-a8ff-afceaeaa6103}",
                "hidden": false,
                "name": "dwhacc",
                "tabTitle": "DWHACC WHYNOT"
            },

**Reply by zadjii-msft** [reply]
@btenden Which `ssh` are you using? `ssh.exe` that ships with Windows? `ssh` from a WSL distro? Or some other `ssh`? And which version of `ssh`? How are you launching the Terminal? 
* Directly from the Start Menu / Run Dialog / `wt.exe`?
* or are you launching `ssh [args...]` and letting it create a terminal for you?

(this subthread may need to get promoted out to it's own issue)

**Reply by btenden** [reply]
@zadjii-msft I'm using ssh from Windows (OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2, from /c/Windows/System32/OpenSSH/ssh)
I'm launching the terminal directly from the start menu.
And then I click on the profile dwhacc.
My OS is Windows10 21H2.

I can change the tabtitle for Powershell/Command Prompt and that works fine.
But only for ssh the changed tabtitle does not show

D:\>which ssh
/c/Windows/System32/OpenSSH/ssh
